src/rsz/src/Resizer.cc

```diff
@@ -2203,6 +2392,9 @@ PinSet* Resizer::findFloatingPins()
 
 ////////////////////////////////////////////////////////////////
 
+//
+// Todo update this to apply over whole design
```

Comment:
I don't know what this refers to.

Reply:
Thanks, I have updated the comment to make it clearer.

The thinking is as follows:
Note how the resizer makes "unique names" & the code defaults to the top instance (see value of top_inst) for the scope of the unique name. But what happens if we want a unique name in a child block somewhere which could (in theory) have a name which clashes with a name in the parent.

So this is a latent bug in the system which needs to be addressed in another pull request. (I was hesitant to put that in this pull request as it could change all the names -- note that because we are running in flat mode everything is ok now, but it definitely needs to be addressed later when hierarchy is default).